docs/articles/high-availability.md

```diff
@@ -0,0 +1,40 @@
+# Deploying a Highly Available zot Registry
+
+> :point_right: A highly available zot registry can be easily implemented using zot's registry synchronization feature.
+
+In the zot configuration, the `sync` extension allows a zot instance to mirror another zot instance with various container image download policies, including on-demand and periodic downloads. You can use the zot `sync`  function combined with a load balancer such as [HAProxy](https://www.haproxy.com) to implement a highly available registry.
+
+Two failover configurations are possible:
+
+* Active/active
```

Comment:
Would make active/active as second option.

Reply:
Swapped order of A/S and A/A in the intro and in the configuration sections